src/pages/kb/data-sources/querying-urls.md

```diff
@@ -6,30 +6,16 @@ slug: json
 toc: true
 ---
 
-Sometimes you need to visualize data not contained in an RDBMS or NOSQL data store. For those times, Redash provides two URL-based mechanisms to pull in your data: `URL` and `JSON`. Either source can be added from the settings screen in Redash.
+Sometimes you need to visualize data not contained in an RDBMS or NOSQL data store, but available from some HTTP API. For those times, Redash provides the `JSON` data source (first introduced in Redash v8).
 
-{% callout info %}
+Redash treats all incoming data from the `JSON` data source as text; so you should be prepared to use 
 
-The `JSON` data source is currently only available as part of SaaS Redash. This page will be updated when it is added to the Open Source version as well. This is expected in the next major version release (version 8).
-
-{% endcallout %}
-
-With `URL`, you can load large amounts of tabular data while abstracting away the underlying storage mechanism. This data source allows you to explicitly declare the types of data returned (text, dates, numbers e.g.).
-
-{% callout warning %}
-The features of the `URL` data source will eventually be rolled into the `JSON` data source. At that time, the `URL` data source will be deprecated and this documentation will be updated. For future development we recommend using the `JSON` data source.
-{% endcallout %}
-
-Now you can use the `JSON` data source for tabular data, non-tabular data, or API's where you do not control the underlying data structure. Redash treats all incoming data from the `JSON` data source as text; so you should be prepared to use [table formatting]({% link _kb/user-guide/visualizations/table-visualizations.md %}) when rendering the data.
```

Comment:
The comment about everything being a string is not accurate: see what I wrote instead below.